lib/main.rb

```diff
@@ -0,0 +1,10 @@
+require_relative 'game'
+
+game = Game.new()
```

Comment:
A better way to instantiate objects here:

player1 = Player.new(<player parameters needed>)
player2 = Player.new(<player parameters needed>)
board = Board.new
game = Game.new(player1, player2, board)
These are related to my comments about just passing instantiated player and board objects into Game rather than instantiating those inside game.